### PR TITLE
chore(seed): freeCodeCampify remaining sections of codebase

### PR DESCRIPTION
**Pull request by no-stack-dub-sack**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

#### Description

This PR is a partial completion of issue #12472.

This mainly targets `seed` files, but I ran a search when I was finished with that directory and found a couple of nagging left over little buggers in other areas of the codebase, so I went ahead and fixed those as well.

I've replaced every instance (visible to the user) of `Free Code Camp`, `free code camp`, `FreeCodeCamp`, `freecodecamp`, and `FREE CODE CAMP` with the much more highly desirable `freeCodeCamp` - except in irrelevant URLs not visible to the user or where required by challenges (ex. regex challenges include other cases in  tests). I did hit some URLs (only the ones that we've changed, i.e. github, codepen), and, I even got the issue template! 

I did mention in #12472 that I did leave a few instances of `freecodecamp` in the Twitch challenge, because not sure how the API would interpret the case change. Also, our name has not yet been changed on Twitch, and we may have left that out of the change plan (looks like we're still FreeCodeCampin' over there! 😜  ).

Anyway, one step closer to putting this issue to rest finally!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -14,18 +14,16 @@
...
 
 #### Your Code
 <!-- If relevant, paste all of your challenge code in here -->
 ```js
- 
- 
- 
+
+
+
```
Comment (Greenheart) [inline .github/ISSUE_TEMPLATE.md]: I would like an space after each bullet here

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json [hunk 2]
```diff
@@ -4057,7 +4057,7 @@
         "The above outputs \"Hello\" to the console, returns \"World\", but <code>\"byebye\"</code> is never output, because the function exits at the <code>return</code> statement.",
         "<hr>",
         "Modify the function <code>abTest</code> so that if <code>a</code> or <code>b</code> are less than <code>0</code> the function will immediately exit with a value of <code>undefined</code>.",
-        "<strong>Hint</strong><br>Remember that <a href='http://www.freecodecamp.com/challenges/understanding-uninitialized-variables' target='_blank'><code>undefined</code> is a keyword</a>, not a string."
+        "<strong>Hint</strong><br>Remember that <a href='http://www.freeCodeCamp.com/challenges/understanding-uninitialized-variables' target='_blank'><code>undefined</code> is a keyword</a>, not a string."
```
Comment (Greenheart) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json]: Do these really need to be changed? Maybe they do actually... :smile:

File: seed/challenges/05-apis-and-microservices/basic-node-and-express.json [hunk 3]
```diff
@@ -234,7 +234,7 @@
         "<code>route_path: '/user/:userId/book/:bookId'</code>",
         "<code>actual_request_URL: '/user/546/book/6754' </code>",
         "<code>req.params: {userId: '546', bookId: '6754'}</code>",
-        "Build an echo server, mounted at the route GET /:word/echo. Respond with a json document, taking the structure {echo: <word>}. You can find the word to be repeated at req.params.word. You can test your route from your browser the address bar, visiting some matching routes, e.g. <your-app-rootpath>/freecodecamp/echo"
+        "Build an echo server, mounted at the route GET /:word/echo. Respond with a json document, taking the structure {echo: <word>}. You can find the word to be repeated at req.params.word. You can test your route from your browser the address bar, visiting some matching routes, e.g. <your-app-rootpath>/freeCodeCamp/echo"
```
Comment (Greenheart) [inline seed/challenges/05-apis-and-microservices/basic-node-and-express.json]: for this URL, it looks cleaner with `freecodecamp`

File: seed/challenges/05-apis-and-microservices/managing-packages-with-npm.json [hunk 4]
```diff
@@ -97,7 +97,7 @@
         "As you can see, this field is structured as an array of double-quoted strings.",
         "Instructions",
         "Add an array of suitable strings to the keywords-field in the package.json of your HyperDev project.",
-        "One of the keywords should be freecodecamp."
+        "One of the keywords should be freeCodeCamp."
```
Comment (Greenheart) [inline seed/challenges/05-apis-and-microservices/managing-packages-with-npm.json]: I think npm keywords are lowercase

File: seed/challenges/05-apis-and-microservices/managing-packages-with-npm.json [hunk 5]
```diff
@@ -110,8 +110,8 @@
           "testString": "getUserInput => $.get(getUserInput('url') + '/_api/package.json').then(data => { var packJson = JSON.parse(data); assert.isArray(packJson.keywords, '\"keywords\" is not an array'); }, xhr => { throw new Error(xhr.responseText); })"
         },
         {
-          "text": "\"keywords\" should include \"freecodecamp\"",
-          "testString": "getUserInput => $.get(getUserInput('url') + '/_api/package.json').then(data => { var packJson = JSON.parse(data);     assert.include(packJson.keywords, 'freecodecamp', '\"keywords\" does not include \"freecodecamp\"'); },  xhr => { throw new Error(xhr.responseText); })"
+          "text": "\"keywords\" should include \"freeCodeCamp\"",
```
Comment (Greenheart) [inline seed/challenges/05-apis-and-microservices/managing-packages-with-npm.json]: Should probably be lowercase

File: seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json [hunk 6]
```diff
@@ -22,8 +22,8 @@
         "text": "\"keywords\" field should be an Array",
         "testString": "getUserInput => ($.get(getUserInput('url') + '/_api/package.json').then(function(data){ var packJson = JSON.parse(data); assert.isArray(packJson.keywords); }, err => { throw new Error('Err: ' + err.statusText);}))"
       }, {
-        "text": "\"keywords\" should include \"freecodecamp\"",
-        "testString": "getUserInput => ($.get(getUserInput('url') + '/_api/package.json').then(function(data){ var packJson = JSON.parse(data); assert.include(packJson.keywords, 'freecodecamp'); }, err => { throw new Error('Err: ' + err.statusText); }))"
+        "text": "\"keywords\" should include \"freeCodeCamp\"",
```
Comment (Greenheart) [inline seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json]: Should probably be lowercase

File: seed/challenges/05-apis-and-microservices/managing-packages-with-npm.json [hunk 7]
```diff
@@ -110,8 +110,8 @@
...
+          "testString": "getUserInput => $.get(getUserInput('url') + '/_api/package.json').then(data => { var packJson = JSON.parse(data);     assert.include(packJson.keywords, 'freeCodeCamp', '\"keywords\" does not include \"freeCodeCamp\"'); },  xhr => { throw new Error(xhr.responseText); })"
```
Comment (Greenheart) [inline seed/challenges/05-apis-and-microservices/managing-packages-with-npm.json]: lowercase

**Review by Greenheart — APPROVED** [review]
@no-stack-dub-sack Thanks for the fix! Generally looking good, I'm just unsure about a few parts that might need to be lowercase either to pass tests, or to look nice as printed URL:s ^^

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (no-stack-dub-sack) [inline .github/ISSUE_TEMPLATE.md]: @Greenheart oddly enough, I did not make a change here. I've noticed this recently in my PRs, something is automatically cleaning up white space.

File: seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json (hunk 6 above)
Comment (no-stack-dub-sack) [inline seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json]: @Greenheart my lack of backend knowledge really shining through here 😄

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json (hunk 2 above)
Comment (no-stack-dub-sack) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json]: No, I don't think so actually. Must have gotten caught in the mix. I don't think it should matter either way though on this one

**Comment by no-stack-dub-sack** [comment]
@Greenheart Ok, check it out when you get the chance. Made requested changes! Off to node school for me 😄

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Greenheart) [inline .github/ISSUE_TEMPLATE.md]: It still seems like there aren't any spaces here though.

**Review by Greenheart — CHANGES_REQUESTED** [review]
Yo gimme some space in the ISSUE_TEMPLATE bro! :neckbeard:

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (no-stack-dub-sack) [inline .github/ISSUE_TEMPLATE.md]: @Greenheart There's 3 new lines between the backticks, no? Not sure why there should be a white space left there? Is there a reason why code shouldn't start at the first character of the new line?

**Comment by no-stack-dub-sack** [comment]
@Greenheart There's 3 new lines between the backticks, no? Not sure why there should be a white space left there? Is there a reason why code shouldn't start at the first character of the new line?

**Comment by Greenheart** [comment]
@no-stack-dub-sack Sorry, I meant a space **after the bullets** like `+ `.

In GitHub markdown, there's a difference between using a space or not:

+hello       (`+hello`)
+ hello      (`+ hello`)

**Comment by no-stack-dub-sack** [comment]
@Greenheart Sure, but this is between sets of  \`\`\`, in which markdown is not interpreted:

```
__ignored markdown__
```

so the only difference between leaving a trailing white space there would be this:

```js
assert(someSuperCoolCode === true, 'message...');
```

vs.

```js
 assert(someSuperCoolCode === true, 'message...');
```
Just a white space 😄 

I'll change if you'd like, I think perhaps I'm still missing the point though

**Comment by Greenheart** [comment]
@no-stack-dub-sack OH! Hahaha I interpreted the `+` signs as bullets in MD...

This is ready :laughing:

**Review by Greenheart — APPROVED** [review]

**Comment by no-stack-dub-sack** [comment]
@Greenheart hahaha, I sensed there was a disconnect somewhere! Well, let me just fix the newlines after the screenshot section before you merge

**Comment by no-stack-dub-sack** [comment]
@Greenheart Okie dokie, once travis is done doing his thing, we're good to go. Phwewf!

Travis - who is that guy anyway? Always building things

**Pull request merged**